include/eve/detail/kumi.hpp

```diff
@@ -799,12 +799,12 @@ namespace kumi
 
   namespace result
   {
-    template<product_type T, product_type... Tuples> struct cat
+    template<product_type... Tuples> struct cat
```

Comment:
why this change?

Reply:
Because otherwise unpacking didb't work.  I think it's a clang bug.